vignettes/unit_test_guidance.Rmd

```diff
@@ -105,14 +105,39 @@ Don’t forget to add a unit test for each exported function.
 
 ## Set up the Test Script
 Within {admiral} folder https://github.com/Roche-GSK/admiral/tree/master/tests/testthat, 
-add a script with the naming convention “test-<function name>.R” and use the 
-following format:
+add a script with the naming convention “test-\<script_containing_function\>.R”.,
+the unit test script can be created from the console also, as follows: 
 
 ```
+usethis::use_test("<script_containing_function>")
+```
+and use the following format:
+
+```
+test_that("<function_name> Test 1: <Explanation of the test>", {
+  input <- tibble::tribble(
+    ~inputvar1, ~inputvar2, ...
+    <Add Test Data Scenarios>
+    ...
+  )
+
+  expected_output <- mutate(input, outputvar = c(<Add Expected Outputs>))
 
-context("test-<function name>")
+  expect_equal(<function name>(input), expected_output)
```

Comment:
When comparing **datasets** we use `expect_dfs_equal()`. That should explicitly called out.

Reply:
I have added a note stating this, let me know if you needed more detail added.